src/tarballs/bin.ts

```diff
@@ -86,5 +80,10 @@ fi
 `, {mode: 0o755})
   }
 
-  await Promise.all([writeWin32(), writeUnix()])
+  await Promise.all([writeWin32(config.bin), writeUnix()])
+  if (process.platform === 'win32') {
```

Comment:
could you concat these other writes onto the array that's going to Promise.all (they'd need to be promisified native exec instead of shelljs) ?